ocfweb/main/templates/home.html

```diff
@@ -26,7 +26,14 @@
 {% block content %}
 <div class="row">
     <div class="col-sm-8 ocf-content-block">
+
         {{lab_status.banner_html | safe}}
+
+        <a href="{% url 'doc' 'services/lab' %}" class="update-banner">
+            <h4>Lab hours have been extended!</h4>
+            <p>Click here to see the updated hours.</p>
+        </a>
+
```

Comment:
I thought the point was to decouple actual development from temporary updates.

We could put multiple banners into lab_status.yaml if we need to have an urgent message and a long term message, for example.